custom_components/zwave_mqtt/binary_sensor.py

```diff
@@ -36,3 +80,101 @@ class ZWaveBinarySensor(ZWaveDeviceEntity, BinarySensorDevice):
     def is_on(self):
         """Return if the sensor is on or off."""
         return self.values.primary.value
+
+    @property
+    def entity_registry_enabled_default(self) -> bool:
+        """Return if the entity should be enabled when first added to the entity registry."""
+        # Legacy binary sensors are phased out (replaced by notification sensors)
+        # Disable by default to not confuse users
+        return False
+
+
+class ZWaveListSensor(ZWaveDeviceEntity, BinarySensorDevice):
+    """Representation of a ZWaveListSensor translated to binary_sensor."""
+
+    @property
+    def is_on(self):
+        """Return if the sensor is on or off."""
+        return self.values.primary.value["Selected"] != "Clear"
+
+    @property
+    def state_attributes(self):
+        """Return the device specific state attributes."""
+        return {"event": self.values.primary.value["Selected"]}
+
+    @property
+    def device_class(self):
+        """Return the class of this device, from component DEVICE_CLASSES."""
+        if self.values.primary.index == 1:
```

Comment:
Break out a module level constant dict that maps index to device class, and look that dict up here.

Reply:
done